### DISCUSSION TITLE
Update Sign Command Wiki

### DISCUSSION THREAD

**Discussion by SophieFox23** [EssentialsX/Essentials#4949] [Ideas and Feature Suggestions] [answered]
Hi there, I noticed that on yalls Wiki for sign commands under the Trade command it says item ID's can still be used when it is no longer an option to be used today. As well as maybe add a disclaimer for people that (as of right now) enchanted books/items are not tradeable/sell/buyable.
![Screenshot (656)](https://user-images.githubusercontent.com/83872862/173950536-ab863a78-52ad-40a7-8149-d6dd586853ce.png)

**Comment by osfanbuff63** [comment, marked as answer]
Most likely caused by the fact that we just redirect to the old `wiki.mc-ess.net` site, and we don't maintain it. Honestly with how long it's been we should probably add a page for i ~~- wait I was doing that in EssentialsX/Wiki#23 lol~~ wait no I just linked to the `wiki.mc-ess.net` site, we should redo it as our own page lol. (see [#1 (comment)](https://github.com/EssentialsX/Wiki/issues/1#issuecomment-570790614))
